media-libs/jasper/jasper-2.0.33.recipe

```diff
@@ -0,0 +1,105 @@
+SUMMARY="Implementation of the codec specified in the JPEG-2000 Part-1 standard"
+DESCRIPTION="The JasPer Project is an open-source initiative to provide a \
+free software-based reference implementation of the codec specified in the \
+JPEG-2000 Part-1 standard. More details about this software can be found in \
+the JasPer Software Reference Manual."
+HOMEPAGE="http://www.ece.uvic.ca/~frodo/jasper/"
+COPYRIGHT="2001-2021 Michael D. Adams"
+LICENSE="JasPer v2"
+REVISION="1"
+SOURCE_URI="https://github.com/mdadams/jasper/archive/version-$portVersion.tar.gz"
+CHECKSUM_SHA256="38b8f74565ee9e7fec44657e69adb5c9b2a966ca5947ced5717cde18a7d2eca6"
+SOURCE_DIR="jasper-version-$portVersion"
+
+ARCHITECTURES="x86_64"
+SECONDARY_ARCHITECTURES="?x86"
```

Comment:
So now you disabled 32bit completely here?

Reply:
My focus is to update x64 first with this merge.

Reply:
> My focus is to update x64 first with this merge.

It shouldn't :) (same for less)

Reply:
Readded x86. Not gcc2!